Source/WTF/wtf/SuspendableWorkQueue.cpp

```diff
@@ -55,16 +74,15 @@ void SuspendableWorkQueue::suspend(Function<void()>&& suspendFunction, Completio
 
     m_state = State::WillSuspend;
     // Make sure queue will be suspended when there is no task scheduled on the queue.
-    WorkQueue::dispatch([this] {
-        suspendIfNeeded();
-    });
+    dispatch([]() { });
 }
 
 void SuspendableWorkQueue::resume()
 {
     ASSERT(isMainThread());
     Locker suspensionLocker { m_suspensionLock };
 
+    RELEASE_LOG_IF(m_shouldLog, SuspendableWorkQueue, "%p - SuspendableWorkQueue::resume current state %s", this, stateString(m_state));
```

Comment:
This needs to use:
`current state %" PUBLIC_LOG_STRING`

Otherwise, it won't show in non-internal sysdiagnoses.